src/main/java/org/opensearch/knn/index/codec/KNN950Codec/KNN950PerFieldKnnVectorsFormat.java

```diff
@@ -25,4 +26,9 @@ public KNN950PerFieldKnnVectorsFormat(final Optional<MapperService> mapperServic
             (maxConnm, beamWidth) -> new Lucene95HnswVectorsFormat(maxConnm, beamWidth)
         );
     }
+
+    @Override
+    public int getMaxDimensions(String fieldName) {
```

Comment:
can you please add a comment that actual field name is ignored and we return a constant.